python/docs/source/examples/bluesky.md

```diff
@@ -1,51 +1,74 @@
 # Bluesky Firehose Example
 
-Bluesky is a "distributed social network" that aims to improve on some of the perceived shortcomings of X (nee Twitter). 
-Bluesky uses a distributed protocol name the [AT Protocol](https://atproto.com/) to exchange messages between users, and provides a "firehose" delivering every message sent over the protocol in real-time. 
+Bluesky is a "distributed social network" that aims to improve on some of the perceived shortcomings of X (nee Twitter).
+Bluesky uses a distributed protocol name the [AT Protocol](https://atproto.com/) to exchange messages between users, and provides a "firehose" delivering every message sent over the protocol in real-time.
 
 In this example, we'll show how you can receive and process the firehose using Kaskada.
 
 You can see the full example in the file [bluesky.py](./bluesky.py)
 
+Othre [bluesky.py](https://github.com/kaskada-ai/kaskada/blob/main/python/docs/source/examples/bluesky.py).
+
 ## Setup the event data source
 
-Before we can receive events from Bluesky, we need to create a data source to tell Kaskada how to handle the events. 
+Before we can receive events from Bluesky, we need to create a data source to tell Kaskada how to handle the events.
 We'll provide a schema and configure the time and entity fields.
 
-.. literalinclude:: bluesky.py
-    :language: python
-    :linenos: 24-50
+```{literalinclude} bluesky.py
```

Comment:
Myst (the way to write these using Markdown instead of RST) uses a different syntax for directives. This is equivalent to the `.. literalinclude:: ...` thing from RST.